arkouda/strings.py

```diff
@@ -2343,6 +2417,10 @@ def register(self, user_defined_name: str) -> Strings:
         """
         self.entry.register(user_defined_name)
         self.name = user_defined_name
+        if self._get_bytes is not None:
+            self._get_bytes.register(self.name + "_get_bytes")
+        if self._get_offsets is not None:
+            self._get_offsets.register(self.name + "_get_offsets")
```

Comment:
just noting so we don't miss it here as well that we don't need `get` in the name.